geomesa-accumulo/geomesa-accumulo-tools/bin/setup-namespace.sh

```diff
@@ -124,11 +129,16 @@ hadoop fs -copyFromLocal -f "$GEOMESA_JAR" "${NAMESPACE_DIR}/${ACCUMULO_NAMESPAC
 
 if hadoop fs -ls "${NAMESPACE_DIR}/${ACCUMULO_NAMESPACE}/geomesa*.jar" > /dev/null 2>&1
 then
+    if [[ -z "$USING_TOKEN" ]]; then
+        ACCUMULO_SHELL_CMD="accumulo shell -u $ACCUMULO_USER -p $ACCUMULO_PASSWORD"
+    else
+        ACCUMULO_SHELL_CMD="accumulo shell -u $ACCUMULO_USER"
+    fi
     echo -e "createnamespace ${ACCUMULO_NAMESPACE}\n" \
       "grant NameSpace.CREATE_TABLE -ns ${ACCUMULO_NAMESPACE} -u $ACCUMULO_USER\n" \
       "config -s general.vfs.context.classpath.${ACCUMULO_NAMESPACE}=${HDFS_URI}${NAMESPACE_DIR}/${ACCUMULO_NAMESPACE}/.*.jar\n" \
       "config -ns ${ACCUMULO_NAMESPACE} -s table.classpath.context=${ACCUMULO_NAMESPACE}\n" \
-      | accumulo shell -u $ACCUMULO_USER -p $ACCUMULO_PASSWORD
+      | ${ACCUMULO_SHELL_CMD}
```

Comment:
Not quite sure what you mean? The difference here is only whether "accumulo shell" is called with a username and password (non-Kerberos) or just the username (Kerberos, assuming cached TGT)